lib/ansible/config/base.yml

```diff
@@ -513,29 +513,6 @@ DEFAULT_CACHE_PLUGIN_PATH:
   ini:
   - {key: cache_plugins, section: defaults}
   type: pathspec
-CALLABLE_ACCEPT_LIST:
```

Comment:
Should removal of this be mentioned in the changelog and/or porting guide?

Reply:
I don't think so. It hasn't be functional since https://github.com/ansible/ansible/pull/20159 which was merged Jan 12, 2017.